pkg/trace/api/profiles.go

```diff
@@ -57,6 +57,9 @@ func profilingEndpoints(apiKey string) (urls []*url.URL, apiKeys []string, err e
 			}
 		}
 	}
+	for i, u := range urls {
+		log.Debug("Set profiling intake URL [%d]:%s", i, u)
```

Comment:
```suggestion
		log.Debugf("Set profiling intake URL [%d]: %s", i, u)
```
I'd expect the linter to catch this.